src/libs/actions/Transaction.ts

```diff
@@ -82,7 +73,8 @@ function saveWaypoint(transactionID, index, waypoint) {
         },
         // Empty out errors when we're saving a new waypoint as this indicates the user is updating their input
         errorFields: {
-            route: null,
+            // TODO: check if its ok to put undefined
```

Comment:
Dev. comment

Reply:
@kubabutkiewicz Why can't we use `null` here?